### DISCUSSION TITLE
blog/judging-open-source-by-github-stars

### DISCUSSION THREAD

**Discussion by giscus** [the-guild-org/the-guild-website#1271] [Docs Discussions]
# blog/judging-open-source-by-github-stars

Exposing the dark side of GitHub and open source projects

https://the-guild.dev/blog/judging-open-source-by-github-stars

**Comment by biplobsd** [comment]
€0.8/star I want to do this 🤣 contact me

**Comment by nene** [comment]
Stars are a pretty crappy metric anyway. Even when the stars are fully legitimate, at best they only indicate how many people have found the project useful. Which might mean various things:

- the project is old and has just gathered lots of stars over many many years.
- the project was popular in the past.
- the project has a good name that comes up easily in search, so many people end up at least trying it.

Unfortunately there's no easy way to judge the quality of the project.
